core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -827,14 +827,26 @@ Please use this method only if you are sure that the software is not working as
 If you have problems or questions, please review the FAQ on the \
 bisq.network web page or post in the Bisq forum in the Support section.
 
-support.initialInfo=Please note the basic rules for the dispute process:\n\
-1. You need to respond to the arbitrator's requests within 2 days.\n\
-2. The maximum period for a dispute is 14 days.\n\
-3. You need to cooperate with the arbitrator and provide the information they request to make your case.\n\
-4. You accepted the rules outlined in the wiki in the user agreement when you first started the application.\n\n\
-Please read more about the dispute process in our wiki:\nhttps://github.com/bisq-network/exchange/wiki/Arbitration-system
+support.initialInfo=Please send a description about your problem in the text field below. \
+  Add as much information as possible to speed up dispute resolution time.\n\n\
+  Here is a check list for information you should provide:\n\
+  \t● If you are the BTC buyer: Did you made the Fiat or Altcoin transfer? If so, did you click the 'payment started' \
+  button in the application?\n\
+  \t● If you are the BTC seller: Did you received the Fiat or Altcoin payment? If so, did you click the 'payment received' \
+  button in the application?\n\
+  \t● Which version of Bisq are you using?\n\
+  \t● Which operating system are you using?\n\
+  \t● If you encountered a software bug with failed transactions please consider to start over with a new data directory.\n\
```

Comment:
Rather than stating "a software bug", perhaps can just generalize it to "an issue"

Reply:
Yes, I would make it "If you encountered an issue with failed transactions please consider switching to a new data directory."

If preferred we could link to the doc instead of the forum (https://docs.bisq.network/backup-recovery.html#switch-to-a-new-data-directory).

Reply:
@m52go Ah I have not seen that. Will use that, but seems there are some important info missing there. Could you compare to the forum link and add those (account age witness -> need to copy sig key).

Reply:
Looks like it's already there? 
https://docs.bisq.network/backup-recovery.html#payment-accounts

Let me know if I'm missing something.